app/models/room.rb

```diff
@@ -13,6 +13,8 @@ class Room < ApplicationRecord
   has_many :frames, through: :bays, dependent: :restrict_with_error
   has_many :materials, through: :frames, dependent: :restrict_with_error
 
+  enum :status, { active: 0, passive: 1, planned: 2 }, default: :active, validate: true
```

Comment:
Move default value in database with the migration. Normally this will be used also as default value in form.